libtransact/src/scheduler/mod.rs

```diff
@@ -164,15 +186,15 @@ mod tests {
 
     pub fn test_scheduler(scheduler: &mut Scheduler) {
         let mut workload = XoBatchWorkload::new_with_seed(5);
-        scheduler.add_batch(workload.next_batch().unwrap());
+        scheduler.add_batch(workload.next_batch().unwrap()).unwrap();
```

Comment:
For these new unwraps, could you change them to expects, so it's clear what's being tested here.

Reply:
Done